api/routes/SavedMeet.js

```diff
@@ -0,0 +1,65 @@
+const express = require('express');
+const router = express.Router();
+const SavedMeet = require('../models/SavedMeet');
+
+// *************************************
+//  SavedMeet-specific routes
+// *************************************
+
+// View all saves
+router.get('/meet/save', (req, res) => {
+  SavedMeet.find()
+  .then(data => res.status(200).send(data))
+  .catch(err => res.send(err))
+})
+// Create a new save
+router.post('/meet/:meetId/save/user/:userId', (req, res) => {
+  SavedMeet.create({
+    userId: req.params.userId,
+    meetId: req.params.meetId,
+    isScheduled: false,
+    name: req.body.name,
+    date_start: req.body.date_start,
+    date_end: req.body.date_end,
+    country: req.body.country,
+    location_name: req.body.location_name || '',
+    street: req.body.street || '',
+    city: req.body.city || '',
+    state: req.body.state || '',
+    zip: req.body.zip || '',
+    cost: req.body.cost || '',
+    url: req.body.url || '',
+  })
+  .then(data => { res.status(200).send(data) })
+  .catch(err => { res.send(err) })
+})
+// View a saves from specific user
```

Comment:
saved* meet
adjusting your comment

Reply:
Fixed, but it is not showing up as a commit.